webcomponents/tool/src/main/frontend/packages/sakai-conversations/web-test-runner.config.mjs

```diff
@@ -0,0 +1,53 @@
+// import { playwrightLauncher } from '@web/test-runner-playwright';
+
+const filteredLogs = ['Running in dev mode', 'lit-html is in dev mode'];
+
+export default /** @type {import("@web/test-runner").TestRunnerConfig} */ ({
+  /** Test files to run */
+  files: 'test/**/*.test.js',
+
+  /** Resolve bare module imports */
+  nodeResolve: {
+    exportConditions: ['browser', 'development'],
+  },
+
+  /** Filter out lit dev mode logs */
+  filterBrowserLogs(log) {
+    for (const arg of log.args) {
+      if (typeof arg === 'string' && filteredLogs.some(l => arg.includes(l))) {
+        return false;
+      }
+    }
+    return true;
+  },
+
+  testRunnerHtml: testFramework =>
+    `<html>
+      <head>
+        <link rel="stylesheet" src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.0.1/css/bootstrap.min.css" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.0.1/js/bootstrap.bundle.min.js"></script>
+      </head>
+      <body>
+        <script>window.process = { env: { NODE_ENV: "development" } }</script>
+        <script type="module" src="${testFramework}"></script>
+      </body>
+    </html>`,
+
+  /** Compile JS for older browsers. Requires @web/dev-server-esbuild plugin */
```

Comment:
Do we need this?

Reply:
We need the test runner file, but if you mean the commented stuff, I just leave that in for doc purposes.

Reply:
Yes, it's about the commented code, if it's good to keep it, i'm fine with it.